libvast/src/system/exporter.cpp

```diff
@@ -73,7 +73,7 @@ void report_statistics(stateful_actor<exporter_state>* self) {
   auto& st = self->state;
   timespan runtime = steady_clock::now() - st.start;
   st.query.runtime = runtime;
-  VAST_INFO(self, "completed in", vast::to_string(runtime));
+  VAST_INFO(self, "relayed", rank(hits), "hits in", vast::to_string(runtime));
```

Comment:
This may not be true, since the hits are a superset of the results. It is only true if the candidate check returns all hits as valid results.

Both information might be helpful: the number of hits as well as the (subset of) relayed results.